charts/incubator/lancache-monolithic/questions.yaml

```
@@ -90,6 +90,35 @@ questions:
                                   schema:
                                     type: int
                                     default: 80
+        - variable: https
+          label: "HTTPS"
+          description: "HTTPS Service"
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+# Include{serviceSelectorSimple}
+# Include{serviceSelectorExtras}
+                    - variable: https
+                      label: "HTTPS"
+                      schema:
+                        additional_attrs: true
+                        type: dict
+                        attrs:
+                          - variable: port
+                            label: "Port"
+                            description: "This port exposes the container port on the service"
+                            schema:
+                              type: int
+                              default: 443
```

Same with the above.
You don't really need the https service tho, as you can use ingress.
```suggestion
                              default: 10280
```

---

I think I need it because I dont want any reverse proxy. Support for the app is best if you just directly pass the port to container it self has a proxy that handels what it needs.

---

Cool, it can stay in either case anyway